### DISCUSSION TITLE
where testable branch

### DISCUSSION THREAD

**Discussion by kazuk** [tokio-rs/tokio#3528] [General] [answered]
I fork and clone tokio-rs/tokio.

`cargo test` failed with `error: build failed` on `master` branch.
and build failed with `tokio-1.2.x` branch.

where branch for testable and can be development.

```
kazuk:~/tokio$ rustc --version
rustc 1.50.0 (cb75ad5db 2021-02-10)
```

**Comment by taiki-e** [comment, marked as answer]
You need to use the `--all-features` argument.

https://github.com/tokio-rs/tokio/blob/master/CONTRIBUTING.md#cargo-commands

> Due to the extensive use of features in Tokio, you will often need to add extra arguments to many common cargo commands. This section lists some commonly needed commands.
> 
> Some commands just need the --all-features argument:
>
> ```
> cargo build --all-features
> cargo check --all-features
> cargo test --all-features
> ```

**Reply by kazuk** [reply]
thanks!, tests under running on my box.
